app/policies/user_context.rb

```diff
@@ -15,4 +15,10 @@ def catalog_access
   def rbac_enabled?
     @rbac_enabled ||= Insights::API::Common::RBAC::Access.enabled?
   end
+
+  def group_uuids
```

Comment:
@eclarizio @hsong-rh 
I am trying to see if the user_context object and spec can follow a similar pattern for both catalog and approval.
If we can change catalog_access to access it might be more generic.
Also allows us to cache the user object down the road without the underlying code needing to be aware of how we got the data.
Also is it possible to validate that all Insights::API::Common::RBAC calls is limited to the user context object.